packages/xo-server/src/xo-mixins/jobs/index.js

```diff
@@ -266,6 +266,13 @@ export default class Jobs {
     })
 
     const app = this._app
+    const user = await app.getUser(job.userId)
+    const data = {
+      callId: Math.random().toString(36).slice(2),
+      userId: user.id,
+      userName: user.name,
+      params: job.vms,
```

Comment:
Params of which method?

Reply:
Indeed, why not integrate the `method` property here.

Reply:
@badrAZ params are id of the vm's selected for the backup job. I got inspired from: https://github.com/vatesfr/xen-orchestra/blob/e0987059d3a9ca1284bee0ce0b2566a1dad51398/packages/xo-server-web-hooks/src/index.js#L92